.github/workflows/ci-e2e-cassandra.yml

```diff
@@ -48,6 +48,13 @@ jobs:
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.major }} ${{ matrix.version.schema }} ${{ matrix.jaeger-version }}
       env:
         SKIP_APPLY_SCHEMA: ${{ matrix.create-schema == 'auto' && true || false }}
+     
+    - name: Upload Metrics Artifact
+      uses: actions/upload-artifact@v3
+      with:
+              name: cassandra-metrics-${{ matrix.version }}
+              path: ./metrics/cassandra_metrics.txt
+              retention-days: 7
```

Comment:
this won't be enough for artifacts uploaded for a release (those should be at 60 days)